### PR TITLE
Sampler priority support (for text-generation-webui)

### PR DESCRIPTION
**Pull request by kalomaze**
Right now, `temperature_last` will take priority over it if it is also on.

image

EDIT: Gave it drag and drop js elements like koboldcpp

image

### CODE REVIEW

**Comment by oobabooga** [comment]
It is also possible to send the request as an array of strings like

```
["temperature", "dynamic_temperature", "quadratic_sampling", "top_k", "top_p", "typical_p", "epsilon_cutoff", "eta_cutoff", "tfs", "top_a", "min_p", "mirostat"]
```

That may help make the request a bit prettier.

**Comment by kalomaze** [comment]
Seems like the load default order for kobold is broken by this PR

File: public/scripts/textgen-settings.js [hunk 1]
```diff
@@ -449,9 +461,37 @@ jQuery(function () {
 
     $('#koboldcpp_default_order').on('click', function () {
         settings.sampler_order = KOBOLDCPP_ORDER;
-        sortItemsByOrder(settings.sampler_order);
+        sortOobaItemsByOrder(settings.sampler_order);
```
Comment (Cohee1207) [inline public/scripts/textgen-settings.js]: This is probably why koboldcpp default order is broken

Comment (kalomaze) [reply public/scripts/textgen-settings.js]: Fixed :D

**Review by Cohee1207 — APPROVED** [review]

**Pull request merged**